source/views/student-orgs/list.js

```diff
@@ -28,7 +28,7 @@ import * as c from '../components/colors'
 import startCase from 'lodash/startCase'
 import Icon from 'react-native-vector-icons/Ionicons'
 import SearchBar from 'react-native-searchbar'
-import type {StudentOrgAbridgedType} from './types'
+import type {GroupingArgsType, StudentOrgAbridgedType} from './types'
 
```

Comment:
Could you just stick this type into the arguments of groupData instead of putting it in another file, please?